src/main/java/org/jabref/gui/maintable/MainTableFieldValueFormatter.java

```diff
@@ -0,0 +1,87 @@
+package org.jabref.gui.maintable;
+
+import java.util.Optional;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.AuthorList;
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.Field;
+import org.jabref.model.entry.field.FieldProperty;
+import org.jabref.model.entry.field.OrFields;
+import org.jabref.preferences.PreferencesService;
+
+import static org.jabref.gui.maintable.MainTableNameFormatPreferences.AbbreviationStyle;
+import static org.jabref.gui.maintable.MainTableNameFormatPreferences.DisplayStyle;
+
+public class MainTableFieldValueFormatter {
+    private final DisplayStyle displayStyle;
+    private final AbbreviationStyle abbreviationStyle;
+    private final BibDatabase bibDatabase;
+
+    MainTableFieldValueFormatter(PreferencesService preferences, BibDatabaseContext bibDatabaseContext) {
+        MainTableNameFormatPreferences nameFormatPreferences = preferences.getMainTableNameFormatPreferences();
+        this.displayStyle = nameFormatPreferences.getDisplayStyle();
+        this.abbreviationStyle = nameFormatPreferences.getAbbreviationStyle();
+        this.bibDatabase = bibDatabaseContext.getDatabase();
+    }
+
+    /**
+     * Format fields for {@link BibEntryTableViewModel}, according to user preferences and with latex translated to
+     * unicode if possible.
+     *
+     * @param fields the fields argument of {@link BibEntryTableViewModel#getFields(OrFields)}.
+     * @param entry the BibEntry of {@link BibEntryTableViewModel}.
+     * @return The formatted name field.
+     */
+    public String formatFieldsValuesLatexFree(final OrFields fields, final BibEntry entry) {
```

Comment:
I think the suffix `LatexFree` can be removed again (since this is implicitly included in "format")

Reply:
Sure! The prefix is removed from the methods of `MainTableFieldValueFormatter`